rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java

```diff
@@ -100,6 +103,8 @@
 
     private static final int AUTO_LOGIN_TIMER_PERIOD_IN_MS = 1000;
 
+    private ExportToCatalogConfirmWindow exportNodeSourceToCatalogWindow;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "exportNodeSourceToCatalogWindow" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)